install.py

```diff
@@ -85,7 +85,7 @@ def setup_ramalama(bindir, tmp_dir):
     to_file = os.path.join(tmp_dir, from_file)
     download(url, to_file)
     ramalama_bin = os.path.join(bindir, binfile)
-    syspath = "/usr/share/ramalama"
+    syspath = "/usr/local/share/ramalama"
     if sys.platform == "darwin":
         install_mac_dependencies()
```

Comment:
maybe we need another function here, to detect atomic desktops and just use a different path here?

Reply:
Nice granite-code usage :)

I would recommend this fix:

```
    sharedirs = ["/opt/homebrew/share", "/usr/local/share", "/usr/share/ramalama"]
    syspath = next((d for d in sharedirs if os.path.exists(d)), None)
    syspath += "/ramalama"
    if sys.platform == "darwin":
        install_mac_dependencies()
```

for this part of the code, but I think this has to be fixed in multiple places.

But, if we use the logic, use "/opt/homebrew/share" if it exists, use "/usr/local/share" if it exists and then use "/usr/share/ramalama" if it exists. I think it should work everywhere.

After we have added the "/ramalama" dir in this script, everywhere else it's better to check for existence of:

```
    sharedirs = ["/opt/homebrew/share/ramalama", "/usr/local/share/ramalama", "/usr/share/ramalama/ramalama"]
```